src/fibonacciBases.js

```diff
@@ -0,0 +1,48 @@
+function convertToFib() {
+  let number = document.getElementById('number').value
+  let result = []
+  let fibSeq = []
+
+//calculate fibSeq value
+  function fibValue(resultArr){
+    let value = 0
+    for(let i = 0; i < fibSeq.length; i++) {
+      value += ( resultArr[i] * fibSeq[i] )
+    } return value
```

Comment:
The `return` statement should be on its own line.  Consider writing code like you would write an essay: each function should have:
```js
const fibonacciValue = resultArray => {
  // The "introductory paragraph", where you handle any set up for the function
  let value = 0

  // Whitespace, followed by the "body" - in an essay, we would have self contained paragraphs,
  // where the paragraph centered around a single point or theme.  In code, we should do the same,
  // and each "paragraph" should be separated by whitespace.
  for( let index = 0; index < calculatedSequence.length; index++ ) {
    value += resultArray[ index ] * calculatedSequence[ index ]
  }
  
  // Finally, our essay's conclusion, typically just the return statement in code
  // Separated from the previous "paragraph" by whitespace
  return value
}
```
Note that I also renamed the variables to help provide the reader with an understanding of their intent.